app/src/main/java/com/example/androidreduxsample/ui/screen/MainScreen.kt

```diff
@@ -76,30 +101,27 @@ fun ArticleList(articles: List<Article>) {
 
 @Composable
 fun ArticleItem(article: Article, onArticleClick: (Int) -> Unit) {
+    val imagePainter = rememberImagePainter(article.urlToImage)
```

Comment:
we lifted this up at func() level, any reason ?

Reply:
Better readability